src/index.html

```diff
@@ -154,13 +154,15 @@ <h4 class="title is-4">
                                 <div class="columns is-gapless">
                                     <div class="field column is-one-fifth">
                                         <label for="allow-adaptations-switch">No</label>
-                                        <input id="allow-adaptations-switch" type="checkbox" name="allow-adaptations-switch" class="switch is-thin" checked="checked">
+                                        <input id="allow-adaptations-switch" type="checkbox" 
+                                        name="allow-adaptations-switch" class="switch is-thin" 
+                                        checked="checked" onchange="switch_callback(this)">
                                         <label for="allow-adaptations-switch">Yes</label>
                                     </div>
                                     <div class="field column is-pulled-left" style="display: inline;">
                                         <p class="control is-pulled-left">
-                                            <div class="b-checkbox is-primary" style="display: inline;">
-                                                <input id="checkbox" class="styled" checked type="checkbox">
+                                            <div class="b-checkbox is-primary" id="sa-checkbox" style="display: inline;">
+                                                <input id="checkbox" class="styled" type="checkbox" onchange="sa_check_callback(this)">
                                                 <label for="checkbox">
                                                     SA Checkbox
```

Comment:
Do you know what you're going to replace this text with?

Reply:
Addressed in commit #fcebb45